docs/src/main/tut/applicative.md

```diff
@@ -43,6 +43,6 @@ into the other context:
 Applicative functors are a generalization of Monads thus allowing to express 
 effectful computations into a pure functional way.
 
-Applicative functors are generally preferred to monads when the structure 
+Applicative functors are generally preferred to Monads when the structure 
```

Comment:
I need some more info to be able to standardize properly, but happy to go through docs once they are clear to me. For now, I changed both to lowercase.

I was referring to e.g. this sentence which uses uppercase a number of times but doesn't use the code style:
```
Like [Functor](functor.md) and [Apply](apply.md), Applicative
functors also compose naturally with other Applicative functors. When
you compose one Applicative with another, the resulting `pure`
operation will lift the passed value into one context, and the result
into the other context:
```